src/application/plugin.rs

```diff
@@ -1,16 +1,21 @@
 use anyhow::Result;
+use directories::ProjectDirs;
 use libset::format::FileFormat;
 use libset::project::Project;
 use proto_rust::provider::provider_client::ProviderClient;
 use proto_rust::provider::{Empty, List};
+use reqwest::Client;
 use serde::{Deserialize, Serialize};
+use std::io::Write;
+use std::os::unix::prelude::PermissionsExt;
+use std::path::PathBuf;
 use std::process::Command;
 use sysinfo::{ProcessExt, System, SystemExt};
 use tonic::transport::Channel;
 
-pub const PLUGINS_URL: &str = "https://raw.githubusercontent.com/done-devs/done/beta/dev.edfloreshz.Done.Plugins.json";
+pub const PLUGINS_URL: &str = "https://raw.githubusercontent.com/done-devs/done/feat-install-plugin/dev.edfloreshz.Done.Plugins.json";
```

Comment:
I think this should target the beta branch again so it continues to work after the merge.

Reply:
Fixed 🎉